src/processing/builder.rs

```diff
@@ -418,6 +453,19 @@ impl<'a> PresentationBuilder<'a> {
             Ok(comment) => comment,
             Err(error) => return Err(BuildError::CommandParse { line: source_position.start.line + 1, error }),
         };
+
+        if let Some(SpeakerNotesMode::Receiver) = self.options.speaker_notes_mode {
+            match comment {
+                CommentCommand::SpeakerNote(note) => {
+                    self.push_text(note.into(), ElementType::Paragraph);
```

Comment:
I initially did this because, for the speaker notes presentation, only these two comments are handled: speaker notes, and end slide. The other comments (e.g. layout, pauses, etc) need not be applied to the speaker notes presentation.

In general, I think my additions to this file need to be refactored.